### PR TITLE
Finalizing changelog for v6.0.0

### PR DESCRIPTION
**Pull request by saimn**
### Description

This pull request is to address ...

Fixes #<Issue Number>

- [ ] By checking this box, the PR author has requested that maintainers do **NOT** use the "Squash and Merge" button. Maintainers should respect this when possible; however, the final decision is at the discretion of the maintainer that merges the PR.

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,3 +1,517 @@
...
+
+- Cosmology I/O now auto-identifies the '.tex' suffix with the 'ascii.latex' format. [#15088]
+
+- The ``Cosmology`` class now has a new property to access the parameters of the
+  cosmology: ``.parameters``. This property return a read-only dictionary of all the
+  non-derived parameter values on the cosmology object. When accessed from the class (not
+  an instance) the dictionary contains ``Parameter`` instances, not the values. [#15168]
+
+- The field ``default`` has been added to ``Parameter``. This can be used to introspect
+  the default value of a parameter on a cosmology class ``e.g. LambdaCDM.H0.default``. [#15400]
```
Comment (nstarman) [inline CHANGES.rst:45]:
```suggestion
  the default value of a parameter on a cosmology class e.g. ``LambdaCDM.H0.default``. [#15400]
```

File: CHANGES.rst [hunk 2]
```diff
@@ -1,3 +1,517 @@
...
+
+- Add ``BaseBodycentricRepresentation``, a new spheroidal representation for bodycentric
+  latitudes and longitudes. [#14851]
+
+- Support Numpy broadcasting over frame data and attributes. [#15121]
+
+astropy.cosmology
+^^^^^^^^^^^^^^^^^
+
+- Added a ``write_latex()`` method for exporting a Cosmology object to a LaTex table. [#14701]
```
Comment (nstarman) [inline CHANGES.rst:29]:
```suggestion
- Registered a ``latex`` writer for exporting a Cosmology object to a LaTex table. [#14701]
```

File: CHANGES.rst (hunk 1 above)
Comment (pllim) [reply CHANGES.rst:45]: Why weren't these caught in the original PR?

File: CHANGES.rst [hunk 3]
```diff
@@ -1,3 +1,517 @@
...
+
+- Add ``WCSAxes.text_coord`` method to print text using ``SkyCoord`` objects
+  parallel to plotting data points with ``WCSAxes.plot_coord``. [#14661]
+
+astropy.wcs
+^^^^^^^^^^^
+
+- Support WCS descriptions of basic planetary coordinate frames. [#14820]
+
+- Updated bundled WCSLIB version to 8.1. This update adds support planetary keywords ``A_RADIUS``, ``B_RADIUS``, ``C_RADIUS``, ``BLON_OBS``, ``BLAT_OBS``, and ``BDIS_OBS`` in ``auxprm`` and adds ``wcsprm::time`` to the ``wcsprm`` struct to record the ``TIME`` axis. This update also includes several bug fixes. For a full list of changes - see http://www.atnf.csiro.au/people/mcalabre/WCS/CHANGES [#15035]
```
Comment (mhvk) [inline CHANGES.rst:135]: Not sure if any editing is wanted, but perhaps good to split this very long line (and, yes, I should have caught that in review...)

**Review by pllim — APPROVED** [review]
Good enough. Let's get the release going. Thanks!

**Pull request merged**